adapters/madvertise/params_test.go

```diff
@@ -50,7 +50,5 @@ var invalidParams = []string{
 	`[]`,
 	`{}`,
 	`{"zoneId":""}`,
-	`{"zoneId":/1111111/banner}`,
```

Comment:
Why were these two test cases removed?

Reply:
This is an invalid param since it's not enclosed in quotes and zoneID is supposed to be a string, no?

Reply:
Yes in fact same `{"zoneId":/1111111}`, I add another test for lenght < 7